battery-status/README.md

```diff
@@ -0,0 +1,6 @@
+Battery Status Sample
+=====================
+
+See https://googlechrome.github.io/samples/battery-status for a live demo.
+
+Learn more at https://www.chromestatus.com/features/4537134732017664
```

Comment:
Can you link to https://www.chromestatus.com/feature/4537134732017664 ("feature", not "features") to go directly to the landing page?